src/Results/Results.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Results from './Results';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Results', () => {
+	let searchResults
```

Comment:
Ill look into if I can get the results added to the testing suite, this is a great start.

Reply:
Thanks! I’ll let you know if I find anything to fix it in the morning as well.